test/distributed/_composable/test_fully_shard.py

```diff
@@ -30,7 +29,13 @@
     SaveForwardInputsModel,
     skip_if_lt_x_gpu,
 )
-from torch.testing._internal.common_fsdp import FSDPTest, TransformerWithSharedParams, CUDAInitMode, FSDPInitMode
+from torch.testing._internal.common_fsdp import (
+    _zero_model,
+    CUDAInitMode,
+    FSDPInitMode,
+    FSDPTest,
+    TransformerWithSharedParams,
+)
```

Comment:
nit: can we have formatting changes in separate PR?

I recognize this is tricky and I think it's time to align on formatting convention for FSDP codebase and automate it. cc @awgu

Reply:
My plan was to just get everyone on `lintrunner` and `lintrunner f` beginning of next half. I decided since we are cranking PRs with urgency right now, we can just not worry about it. The PR to achieve this look like: https://github.com/pytorch/pytorch/pull/90873 I have re-pushed recently, but the main change is just in the `.lintrunner.toml` file and making sure all relevant files are compliant.

I do think that unifying under `lintrunner` / `lintrunner f` is nice. Sometimes I add changes to a file that create long lines or add imports, and I want to just auto-format. However, without an agreed-upon auto-formatter, this becomes a problem and actually complicates the workflow.

Reply:
Will rebase this PR on top of https://github.com/pytorch/pytorch/pull/91255.